src/System.Text.Encodings.Web.Utf8/Utf8BufferEncoder.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Text.Encodings.Web.Buffers;
+
+namespace System.Text.Encodings.Web.Internal
+{
+    internal class Utf8UriEncoder : BufferEncoder
+    {
+        public override OperationStatus Encode(ReadOnlySpan<byte> input, Span<byte> output, out int consumed, out int written)
+        {
+            if (Utf8.UrlEncoder.TryEncode(input, output, out written))
```

Comment:
From above, it states: `[Obsolete("Use UrlEncoder.Utf8.TryEncode")] `
Here, we are using Utf8.UrlEncoder.TryEncode which is a bit confusing. What is the type of `Utf8` here?

Reply:
The type of Utf8 is BufferEncoder